images/build/go-runner/variants.yaml

```diff
@@ -1,12 +1,12 @@
 variants:
-  go1.21-bullseye:
-    CONFIG: 'go1.21-bullseye'
-    IMAGE_VERSION: 'v2.3.1-go1.21.0-bullseye.0'
+  go1.21-bookworm:
```

Comment:
should we keep both for now or that is not needed?

Reply:
I'd say we switch to bookworm with 1.21 completely. Would that work?

Reply:
sgtm, thanks


/lgtm
/approve